sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala

```diff
@@ -89,10 +89,12 @@ class V2SessionCatalog(catalog: SessionCatalog)
           throw QueryCompilationErrors.timeTravelUnsupportedError(
             toSQLId(catalogTable.identifier.nameParts))
         } else {
-          throw QueryCompilationErrors.tableNotSupportTimeTravelError(ident)
```

Comment:
can we remove `tableNotSupportTimeTravelError` from `QueryCompilationErrors`? @panbingkun @beliefer

Reply:
Let me investigate it.

Reply:
Its corresponding error class '_LEGACY_ERROR_TEMP_1337' has been merged with `UNSUPPORTED_FEATURE.TIME_TRAVEL`, so it has been removed here.